jitpack.yml

```diff
@@ -0,0 +1,2 @@
+jdk:
+  - openjdk11
```

Comment:
Is this something that we want to include in this PR? If so, why?

Reply:
We don't need this file. I have removed it.